studio/components/ui/Charts/BarChart/BarChart.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+import { action } from '@storybook/addon-actions';
+import { BarChart } from './BarChart'
+
+export default {
+  title: 'Charts/BarChart',
+  component: BarChart,
+} as ComponentMeta<typeof BarChart>
+
+const Template: ComponentStory<typeof BarChart> = (args) => <BarChart {...args} />
+
+const data = [290, 430, 620, 67, 354, 123, 627, 122].map(val => ({ "ram_usage": val}));
+
+export const Default = Template.bind({})
+const defaultArgs = {
```

Comment:
move the default args into the template and spread them into the component. then let `args` override.
```
<BarChart {...{my: 'attr'}} />
```